### PR TITLE
Cld signed upload

### PR DESCRIPTION
**Pull request by yakirp**
I would like to add this template as Cloudinary signed upload endpoint.
For more information about, please see this:
https://cloudinary.com/documentation/upload_widget#signed_uploads
Thanks
--Yakir

### CODE REVIEW

**Review by guykorol1 — APPROVED** [review]

**Review by guykorol1 — COMMENTED** [review]
.

**Comment by NotMyself** [comment]
Hey, great idea! I appreciate you submitting a new template!

I am curious if express is needed in this webtask at all. There is only a single route, it seems like using the "[Function with context](https://webtask.io/docs/model)" programming model would greatly simplify the code.

**Review by NotMyself — CHANGES_REQUESTED** [review]
Hey, great idea! I appreciate you submitting a new template!

I am curious if express is needed in this webtask at all. There is only a single route, it seems like using the "[Function with context](https://webtask.io/docs/model)" programming model would greatly simplify the code.

**Comment by yakirp** [comment]
Hi, 
Thanks for the feedback! 
I removed Express from the code, and test the new implantation, works well. 

--Yakir

**Review by NotMyself — APPROVED** [review]

File: templates/cloudinary_signed_upload.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+author: 
+  name: yakirp
+  link: https://github.com/yakirp
+type: webhook
+description: |
+  A webtask that can be used as an endpoint for signed upload calls (from upload widget, Android, iOS,. etc)
+note:
+  title: How to use with Cloudinary Upload Widget
+  content: |
+    This template creates endpoint which generate and return signature of the upload request
```
Comment (yavorg) [inline templates/cloudinary_signed_upload.yaml]: Can we add one or two sentences here describing the high-level scenario the widget is used for? Something along the lines of what you have in the docs, but maybe shortened. 
> Cloudinary's upload widget is a complete, interactive user interface that enables your users to upload files from a variety of sources to your website or application. The widget, requiring just a couple lines of code to integrate, eliminates the need to develop in-house interactive media upload capabilities.

Comment (yakirp) [reply templates/cloudinary_signed_upload.yaml]: @yavorg Sure you can! This is a great idea.

Comment (yakirp) [reply templates/cloudinary_signed_upload.yaml]: @yavorg I added the description, a bit long, no?

**Pull request merged**